app/views/keywords/index.html.erb

```diff
@@ -18,41 +18,26 @@
   <% end %>
 
 </div>
-
-<table class="table table-hover">
-  <thead>
-  <tr>
-    <th scope="col">#</th>
-    <th scope="col">Keyword</th>
-    <th scope="col">Created At</th>
-    <th scope="col">Status</th>
-  </tr>
-  </thead>
-  <tbody>
-  <tr>
-    <th scope="row">1</th>
-    <td>Hello World! Ruby on Rails</td>
-    <td>Wed, 1 Dec</td>
-    <td class="font-italic">Searching...</td>
-  </tr>
-  <tr>
-    <th scope="row">2</th>
-    <td><a href="#">Swift</a></td>
-    <td>Wed, 1 Dec</td>
-    <td>Done</td>
-  </tr>
-  <tr>
-    <th scope="row">3</th>
-    <td>Ruby on Rails</td>
-    <td>Wed, 1 Dec</td>
-    <td class="font-italic">Searching...</td>
-  </tr>
-  <tr>
-    <th scope="row">4</th>
-    <td><a href="#">Web Development</a></td>
-    <td>Wed, 1 Dec</td>
-    <td>Done</td>
-  </tr>
-  </tbody>
-</table>
-
+<% if not @decorated_keywords.empty? %>
```

Comment:
@suho We should not use instance variables in views as per our compass conventions https://github.com/mindvalley/platform/pull/3625/commits/78536625a669db5504ef6a8a0d177f78b6c33375.

Reply:
Good catch! Thank you! I will update this

Reply:
Fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695)